config.js

```diff
@@ -146,7 +146,10 @@ config.logging = {
     format: 'pipe',
     // The time in seconds between proof of life logging messages informing that the server is up and running normally.
     // Default value: "60"
-    proofOfLifeInterval: '60'
+    proofOfLifeInterval: '60',
+    // The time in seconds between proof of performance logging messages informing that the server is up and running normally.
+    // Default value: "60"
+    proofOfPerformanceInterval: '60'
```

Comment:
As far as I understand, this is not exactly a "proof of performance" but some statistics about request. Thus, I'd suggest to change the name, for instance "processedRequestLogStatisticsInterval". Code comment should be also adapted to the new name.

Reply:
Similar adaptation for PROOF_OF_PERFORMANCE_INTERVAL env var name.

Reply:
Made changes as per your suggestion. Fixed in 77610b5